packages/block-editor/src/components/block-pattern-picker/layout-setup-toolbar.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Button } from '@wordpress/components';
+import {
+	chevronRight,
+	chevronLeft,
+	grid,
+	stretchFullWidth,
+} from '@wordpress/icons';
+
+function LayoutSetupToolbar( {
+	viewMode,
+	setViewMode,
+	handlePrevious,
+	handleNext,
+	activeSlide,
+	totalSlides,
+	onBlockPatternSelect,
+	onStartBlank,
+} ) {
+	const isSingleView = viewMode === 'single';
```

Comment:
What is the single view mode describing?

Reply:
`single` is when using the carousel which shows one pattern at a time.

Reply:
Might be worth a comment